tailnet/coordinator.go

```diff
@@ -212,6 +224,19 @@ type Queue interface {
 	Close() error
 }
 
+type QueueV2 interface {
```

Comment:
`*connIO` implements it, and I was originally planning to use it internally.  I changed my mind to just passing `*connIO` around, since it's all internal to the coordinator and there is only one implementation.  I think we can just drop the `interface`